src/services/apiService.ts

```diff
@@ -51,6 +51,11 @@ class ApiService {
         return this.client.post(url, qs.stringify(data));
     }
 
+    public testToken = () => {
+        const url = "api/v1/login/test-token";
+        return this.client.post(url);
+    }
+
     public getCurrentUser = () => {
         const url = "api/v1/users/me";
```

Comment:
@mateuszpmroz @bklukaczewski I was wondering if it would not be a better practice to have all the API urls in a config file of some sort, instead of hardcoded in every function. What do you think?

Reply:
Agree 100%. It can be some enum or const, what do you prefer.
Will also need to apply this to AuditApp as it's a really nice improvement.

Reply:
Agree, it's much better to have endpoints in separated file/place